gr-digital/lib/meas_evm_cc_impl.cc

```diff
@@ -0,0 +1,72 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "meas_evm_cc_impl.h"
+#include <gnuradio/io_signature.h>
+#include <numeric>
+
+namespace gr {
+namespace digital {
+
+meas_evm_cc::sptr meas_evm_cc::make(constellation_sptr cons, evm_measurement_t meas_type)
+{
+    return gnuradio::get_initial_sptr(new meas_evm_cc_impl(cons, meas_type));
+}
+
+meas_evm_cc_impl::meas_evm_cc_impl(constellation_sptr cons, evm_measurement_t meas_type)
+    : gr::sync_block("meas_evm_cc",
+                     gr::io_signature::make(1, 1, sizeof(gr_complex)),
+                     gr::io_signature::make(1, 1, sizeof(float))),
+      d_cons(cons),
+      d_cons_points(d_cons->points()),
+      d_meas_type(meas_type)
+{
+    // Calculate the average power of the constellation
+    float d_cons_mag_sq =
```

Comment:
remove the `float`, so that member variable is actually set. That was my mistake in my previous comment. sorry.